sdk/communication/communication-chat/src/generated/src/models/mappers.ts

```diff
@@ -266,7 +266,14 @@ export const SendChatMessageRequest: coreHttp.CompositeMapper = {
       type: {
         serializedName: "type",
         type: {
-          name: "String"
+          name: "Enum",
+          allowedValues: [
+            "text",
```

Comment:
We should have this list in just one place. I suggest we create a variable/static list that is used here and in line 351

Reply:
This is auto generated code. I did a swagger transformation in config file to generate the type into string union instead of string